driver_test.go

```diff
@@ -34,6 +34,7 @@ var (
 	protocol         string
 	customPrivateKey bool            // Whether user has specified the private key path
 	testPrivKey      *rsa.PrivateKey // Valid private key used for all test cases
+	debugMode        bool
```

Comment:
I don't think we need it :) Can you remove it?

Reply:
I was using this to print the full query in driver_test as an override to the `query too large to print` locally. You should only ever get this behavior by setting `SNOWFLAKE_TEST_DEBUG=true`